package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@divriots/starter-simba",
+  "name": "@divriots/simba",
```

Comment:
Hmmm I guess the npm package is changing too then... @georges-gomes is this intended, want me to add npm redirect for starter-simba etc. as well saying we've changed pkg name? Or wanna keep it the same? If we change package, 0.1.0 or 1.0.0 or 0.7.0?

Reply:
I think we should rename the package indeed, but don't we want to prefix with smth, like `ds-simba` or suffix `simba-ds`?